x-pack/plugins/actions/server/builtin_action_types/lib/axios_utils.ts

```diff
@@ -35,16 +32,13 @@ export const request = async <T = unknown>({
   const { httpAgent, httpsAgent } = getProxyAgents(proxySettings, logger);
 
   return await axios(url, {
+    ...rest,
     method,
     data: data ?? {},
-    params,
-    auth,
     // use httpAgent and httpsAgent and set axios proxy: false, to be able to handle fail on invalid certs
     httpAgent,
     httpsAgent,
     proxy: false,
-    headers,
-    validateStatus,
```

Comment:
The way the code was written meant that `validateStatus` was set to `undefined`, instead of not being specified at all. This broke the behaviour of axios to throw an error when a non-200 status code was received after upgrading to axios.